srcpkgs/rust/template

```diff
@@ -98,17 +98,6 @@ else
 	esac
 fi
 
-# In rust terminology 'build' is the build host ($CBUILD) and `target`
-# and `host` are the triplets that are supposed to run the built binaries
-# ($CTARGET)
-case $XBPS_MACHINE in
-	ppc64le) _build_triplet=powerpc64le-unknown-linux-gnu;;
-	ppc64le-musl) _build_triplet=powerpc64le-unknown-linux-musl;;
-	ppc64-musl) _build_triplet=powerpc64-unknown-linux-musl;;
-	*-musl) _build_triplet=${XBPS_MACHINE%-musl}-unknown-linux-musl;;
-	*) _build_triplet=${XBPS_MACHINE}-unknown-linux-gnu;;
-esac
```

Comment:
```
         # Rust target, which differs from our triplets
        export RUST_TARGET="$XBPS_CROSS_RUST_TARGET"
```

Reply:
The thing is, `XBPS_TRIPLET` is always a full triplet (say, `powerpc64le-unknown-linux-gnu`) while `XBPS_CROSS_TRIPLET` is typically a short triplet (say, `powerpc64le-linux-gnu`) in order to match the cross-toolchain name. That is why `RUST_TARGET` is different in a cross environment, as Rust requires a full triplet (with the `unknown` or whatever in it), in non-cross `RUST_TARGET` should match `XBPS_TRIPLET`.

Reply:
OTOH, now that I am looking at it, for `armv6l` and `armv7l` as well as `i686` and `ppc` this is not true. It seems like it could be fixed though